store/local_cache_supplier_schemes.go

```diff
@@ -9,14 +9,6 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 )
 
-func (s *LocalCacheSupplier) handleClusterInvalidateScheme(msg *model.ClusterMessage) {
```

Comment:
It looks like something implemented for future usage, but not used yet. I'm not sure if we want to remove this. @grundleborg maybe you have an strong opinion here. I think is your code.

Reply:
Yeah it's implemented so it's there when needed, as in the other stores. I guess, since it isn't being used we *could* delete it but I'd rather not.

Reply:
@DSchalla can we keep this function then?

Reply:
@jespino @grundleborg Reverted it

Reply:
Thanks!